app/assets/main/react_components/footprint_form/FootprintForm.jsx

```diff
@@ -184,6 +184,7 @@ const FootprintForm = ({ calculator, questionStrings, options, footprint, URL, t
             </div>
           </div>
         }
+        <div id="information-scroll-placeholder"></div>
```

Comment:
```suggestion
        <div id="information-scroll-position"></div>
```
"placeholder" för mig antyder att det ska läggas till/bytas ut där sen, visst är det positionen ba?